query-node/schema.graphql

```diff
@@ -237,45 +281,60 @@ type Video @entity {
   channel: Channel!
 
   "Reference to a video category"
-  category: Category!
+  category: Category
 
   "The title of the video"
-  title: String! @fulltext(query: "search")
+  title: String @fulltext(query: "search")
 
   "The description of the Video"
-  description: String!
+  description: String
 
   "Video duration in seconds"
-  duration: Int!
+  duration: Int
 
   "Video's skippable intro duration in seconds"
   skippableIntroDuration: Int
 
   "Video thumbnail url (recommended ratio: 16:9)"
-  thumbnailUrl: String!
+  thumbnailUrl: String
 
   "Video's main langauge"
   language: Language
 
-  "Reference to VideoMedia"
-  media: VideoMedia!
-
   "Whether or not Video contains marketing"
   hasMarketing: Boolean
 
   "If the Video was published on other platform before beeing published on Joystream - the original publication date"
-  publishedBeforeJoystream: Int
+  publishedBeforeJoystream: DateTime
 
   "Whether the Video is supposed to be publically displayed"
-  isPublic: Boolean!
+  isPublic: Boolean
 
-  "Video curation status set by the Curator"
-  isCurated: Boolean!
+  "Flag signaling whether a video is censored."
+  isCensored: Boolean!
 
   "Whether the Video contains explicit material."
-  isExplicit: Boolean!
+  isExplicit: Boolean
+
+  license: LicenseEntity
+
+  "Encoding of the video media object"
+  codecName: String
```

Comment:
Any specific reason to drop the VideoMedia and add the fields directly here? I personally liked that separation of video metadata and video file metadata

Reply:
We've discussed this with Mokhtar on the call, and if I understood it correctly, the reason was simply not to have 1:1 relations anymore, but maybe I'm mistaken. What do you think @mnaamani?

Reply:
My bad, I had asked Andy to drop the separate entities because I assumed this model was chosen to match explicitly the separate entities in the old content directory. In new content directory there isn't this explicit separation so we can revert if it makes more sense. - sorry @ondratra

Reply:
From Atlas side, it doesn't make that much of a difference, I think it's just better DX separated. If we don't care, let's keep the old version

Reply:
`VideoMedia` was reintroduced, but under the name `VideoMetadata`.

Reply:
Just a thought: maybe we should call this `VideoMediaMetadata` or something else? Generally, I'd expect the title for example to be `VideoMetadata` as well. I think the current name may not properly indicate that we're talking about the media

Reply:
@ondratra ping

Reply:
Sorry, I've missed the last reply. The rename does make sense since other video-related properties can also be considered metadata, and this change better distinguishes them. The name has been changed, and the entity is called `VideoMediaMetadata` now.